src/url-replacements.js

```diff
@@ -14,15 +14,16 @@
  * limitations under the License.
  */
 
-import {assert} from '../src/asserts';
-import {documentInfoFor} from '../src/document-info';
-import {parseUrl, removeFragment} from '../src/url';
+import {assert} from './asserts';
+import {documentInfoFor} from './document-info';
+import {getService} from './service';
+import {parseUrl, removeFragment} from './url';
 
 
 /**
  * This class replaces substitution variables with their values.
  */
-export class UrlReplacements {
+class UrlReplacements {
```

Comment:
It's been intended as composition, not inheritance. If we proceed as a singleton, let's hide "set" method.